README.md

```
@@ -298,8 +298,90 @@ A branch with no committed changes also receives `NO`.
+For a single autonomous review pass use `puny --review` directly. To loop
+`implement → review → fix` until merge worthy, use the external orchestrate
+scripts in `scripts/` (see below).
+
+### Orchestrate a plan → implement → review loop
+
+`scripts/orchestrate.sh` (bash) and `scripts/orchestrate.ps1` (PowerShell) wrap
+`puny --prompt --oneshot` + `puny --review` into the autonomous flow
+`[plan] → implement → (while review != merge worthy) → [fix findings]` and exit
+with the final review code.
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)